app/js/components/LabTrendsPage.jsx

```diff
@@ -109,7 +110,7 @@ export class LabTrendsPage extends PureComponent {
 
     const formatChartData = data => data.map(item => ({
       ...item,
-      obsDatetime: moment(item.obsDatetime).format('DD-MMM-YYYY'),
+      obsDatetime: moment(item.obsDatetime).valueOf(),
```

Comment:
This is related to fixing the the issue with x-axis of the trends graph, in UHM-5350